db/migrate/20220901113232_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
```

Comment:
- [ ] Kindly check on your migration file no  20220826095422, am getting this error when running your app:
![image](https://user-images.githubusercontent.com/61408860/188102368-d508e27e-ddd2-44fb-9632-8167406674ee.png)

- [ ] While on it kindly ensure your design matches the wireframe given I was unable to check on it due to the error above.